webapp/components/search_results.jsx

```diff
@@ -144,6 +149,14 @@ export default class SearchResults extends React.Component {
         });
     }
 
+    onSearchTermChange(doSearch, isMentionSearch) {
+        if (this.mounted && doSearch) {
+            this.setState({
+                loading: true
+            });
+        }
+    }
```

Comment:
When that event is fired, if a search is underway, the search_results panel will know that it is to be in a loading state